source/manual/howto-use-restored-db-in-app.html.md

```diff
@@ -0,0 +1,83 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: How to use a restored database in an app
+section: Backups
+layout: manual_layout
+parent: "/manual.html"
+---
+
+Once you have [restored a database in AWS RDS](/manual/howto-backup-and-restore-in-aws-rds.html),
+you can now use this in your app.
+
+> The steps outlined below are for what to do during an incident where you may need to
+temporarily switch to use a backup. For a more permanent solution this will require
+work in Terraform as any new instances will still use the original database.
```

Comment:
This leads the reader down a different path than intended - we're not suggesting they use Terraform, we're suggesting they restore another DB to the original DB name and then point back to that. I think we can just remove this pullout above.